### DISCUSSION TITLE
Sensor node: after reboot Homeassitant the history of the created sensor is gone?

### DISCUSSION THREAD

**Discussion by askmurphy** [zachowj/node-red-contrib-home-assistant-websocket#1605] [Q&A]
I like the sensor node, easy to create a new sensor In Homeassitant. But, when i make such a sensor and reboot homeassitant after some days the history of the created sensor is lost??  Is this a bug or is there something what must be done to keep the sensor history?

**Comment by zachowj** [comment]
Are you referring to the entire state history of the sensor, or just the most recent state before the reboot?

**Comment by askmurphy** [comment]
most recent state before the reboot..
But I had a lot of other issues, registration in HA works but the data from a node is not allways send to homeassistant too.
After a whole day, I gave up.   What I tried was this: a nodered flow with a TTN (LoRa) mqtt node give me the correct data. Sensor node filled with the importent sensor and also added some attributes.  Sensor is created in HA and I get data. After a reboot, no data. Manual booted the lora-device, new data in HA. Most of the time it works, but the sensor node has connection troubles often..  No time anymore, will use the old fasion method with mqtt node and a yaml template..
